tests/unit/macros/base.py

```diff
@@ -49,5 +49,8 @@ def dispatch(macro_name, macro_namespace=None, packages=None):
 
         self.default_context["adapter"].dispatch = dispatch
 
-        value = getattr(self.template.module, name)(*args)
+        return getattr(self.template.module, name)(*args)
+
+    def _run_macro(self, name, *args):
```

Comment:
I'm not super familiar with the dbt macro system. But this is simply a refactor, right? It doesn't change the behavior of the `_run_macro()` method -- only decomposes such that each function (_run_macro and _run_macro_raw) does just one thing. Right?

Reply:
yes, decomposed because the space stripping is used in every test of test_adapters_macros, but for the test_python_macros tests, I actually care about spacing (because python is like that :/)